readme.md

```diff
@@ -27,7 +27,23 @@ console.log(wrapAnsi(input, 20));
 
 ## API
 
-### wrapAnsi(input, columns)
+### wrapAnsi(input, columns, opts)
+
+wrap words to the specified column width. By default the wrap is
+`soft`, so long words may extend past the column length.
+
+the following options can be provided:
+
+* `hard`: `boolean`, should hard wrapping be enabled?
+
+### wrapAnsi.hard(input, columns)
+
+long words will be broken up so that they do not extend past the column width.
+Equivalent to `{hard: true}`.
+
+### wrapAnsi.soft(input, columns)
```

Comment:
It's four letters; my vote is on separate methods unless the code is pretty much the same (in fairness I haven't looked that hard at it; getting ready to fly out!).